README.md

```diff
@@ -19,6 +19,10 @@ that incorporates and documents best practices for a performant and scalable des
 security by default for control, logging and evidence generation. It can be  simply deployed by
 customers through a Terraform workflow.
 
+## Disclaimer
+
+To ensure a safe environment we highly recommend you remove all owners roles in the projects used as inputs for *Data Warehouse module*, respecting the principle of [separation of duties](https://cloud.google.com/kms/docs/separation-of-duties).
```

Comment:
We can make the readability better.
~To ensure a safe environment~ ... We ~highly~ recommend you remove all primitive `owner` roles...
... This recommendation aligns with the principle of separation of duties...